pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -225,6 +226,11 @@ buildPythonApplication rec {
     runHook postInstall
   '';
 
+  disabledTests = [
+    # Assertion Error on x86_64-darwin
+    "test_transfer_send"
+  ];
```

Comment:
Tests are already being disabled in `preCheck`, please move this there.

Alternatively, I prefer this format. But I'd want to see the check be conditional on darwin, and the checks from `preCheck` moved here maintaining their application to only specific systems. This is more work and will require more testing, so the simpler option is to just move this to the existing disable area in `preCheck`

Reply:
Didn't notice this first time around, was just trying to get Ofborg to run it to see if it would pass before heading to bed as I don't have access to a machine running that architecture.

I'd be happy to work on the required changes to move disabling tests out of `preCheck` and into `disabledTests` in the future. 

For now though, I've moved the disabled test to `preCheck` in https://github.com/NixOS/nixpkgs/pull/256072/commits/d9bd88aaf6c37a46a36f9bf090c72c26123fdcaf